lms/templates/courseware/info.html

```diff
@@ -54,7 +54,7 @@ <h2 class="title">${_("You are not enrolled yet")}</h2>
 <section class="container">
   <div class="home">
     <div class="page-header-main">
-        <h1 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.id.org, course_name=course.id.course) | h}</h1>
+        <h1 class="page-title">${_("Welcome to {org}'s {course_name}!").format(org=course.display_org_with_default, course_name=course.display_number_with_default) | h}</h1>
```

Comment:
You'll want to review the [docs here](http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#mako-template-files) for proper handling of this.  You'll want to adjust the import line, and you can remove the "| h" here. 

Note that this change will affect all calls to `_()`, which is generally good, but you will need to follow the docs above to keep [the following lines](https://github.com/itsbenweeks/edx-platform/blob/info_display_number_fix/lms/templates/courseware/info.html#L24-L26) working properly using the `HTML()` method.